<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>9 - 2D Transformations - Rados≈Çaw Mariowski</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frmCanvas">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sldX">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-200</number>
    </property>
    <property name="maximum">
     <number>700</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sldY">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>90</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-200</number>
    </property>
    <property name="maximum">
     <number>700</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sldScaleX">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>150</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>3000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sldScaleY">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>220</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>3000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sldRotate">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>290</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>720</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sldShearingX">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>360</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>2000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sldShearingY">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>430</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>2000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblX">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblY">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>70</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblScaleX">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>130</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Scale X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblScaleY">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>190</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Scale Y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblRotation">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>260</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Rotation:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblShearingX">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>330</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Shearing X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblShearingY">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>400</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Shearing Y:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReset">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>520</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Resetuj</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
